src/components/character-count/index.md.njk

```diff
@@ -97,6 +97,16 @@ For example, ‘Enter a summary’.
 Say ‘[whatever it is] must be [number] characters or fewer’.<br>
 For example, ‘Summary must be 400 characters or fewer’.
 
+### Consider the effect of emojis and accented characters
```

Comment:
Is there any advice we could embed in this heading title? Like advising our users to tell  _their_ users about emojis and accented characters inflating the count?

Reply:
That's a great suggestion, thanks. Will add in.